core/src/main/java/org/apache/lucene/queryparser/classic/MapperQueryParser.java

```diff
@@ -779,7 +757,10 @@ private Query applySlop(Query q, int slop) {
                 builder.add(terms[i], positions[i]);
             }
             pq = builder.build();
-            pq.setBoost(q.getBoost());
+            //nocommit why do we deal with boost here? isn't that what applyBoost does?
```

Comment:
Yeah! Good point. Maybe applySlop is called after applyBoost? That isn't a good reason, but it might be the reason.